javascript/javascript3/week1/weatherapp/main.js

```diff
@@ -0,0 +1,121 @@
+const input = document.getElementById('input');
+const weatherBtn = document.getElementById('weatherBtn');
+const paraTemp = document.getElementById('paraTemp'); 
+const paraCity = document.getElementById('paraCity'); 
+const paraDesWeather = document.getElementById('paraDesWeather');
+const sunrise = document.getElementById('sunrise'); 
+const sunset = document.getElementById('sunset'); 
+const windSpeed = document.getElementById('windSpeed'); 
+const cloudiness = document.getElementById('cloudiness'); 
+const main = document.getElementById('main'); 
+const icon = document.getElementById('icon'); 
+
+const weatherApp = async () => {
+    const inputArea = input.value;
+    const weatherInfo = await fetchData(inputArea); 
+    const unsplash = await changeBackgroundFunc(inputArea); 
+    const tempIs = Math.round(weatherInfo.main.temp - 273.15);
+    const desWeather = weatherInfo.weather[0].description;
+    const srTime = new Date((weatherInfo.sys.sunrise + weatherInfo.timezone)*1000);
+    const ssTime = new Date((weatherInfo.sys.sunset + weatherInfo.timezone)*1000);
+
+        icon.src = 'http://openweathermap.org/img/wn/'+ weatherInfo.weather[0].icon + '@2x.png';   
+
+        sunrise.style.background = 'rgba(107, 149, 207, 0.396)'; 
```

Comment:
It is suggested to keep styling on the CSS files only. In case you wanted to give some conditional styling, then an option ould be to set a default style in the CSS files and then change it with DOM manipulation only if needed.

Reply:
Hi Martin, 

Thank you for your review :) 
I have fixed my code as you recommended, I am not sure what I fixed in this part is correct?
I removed line 24 - 27 and put background style in CSS stylesheet and set the parent element (.moreInfo) display none, in case when the user is click the button to generate the weather moreInfo element will change to display 'flex'

PS: for my previous code line 24- 27, I did style those elements in JS because if I did it in css stylesheet then it will shows background before the user is click on generate btn which is I dont want it to  show before